lib/fs/copy-symlink.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const nativeFs = require('fs');
+const path = require('path');
+
+const promisify = require('es6-promisify');
+const makeDir = require('make-dir');
+
+/**
+ * Copies symlink to destination.
+ *
+ * @param {string} source Path to symbolic link you want to copy.
+ * @param {string} destination Where you want the symbolic link copied.
+ * @param {Object} [options]
+ * @param {Object} [options.fs] Use a custom fs implementation. For example `graceful-fs`.
+ * @returns {Promise}
+ */
+module.exports = (source, destination, options) => {
+    const opts = options || {};
+    const destPath = path.resolve(destination);
+    const destDir = path.dirname(destPath);
+
+    const fs = opts.fs || nativeFs;
+    const readlink = promisify(fs.readlink);
```

Comment:
May be we can move it to top of this module? Now the `promisify` function will be called for each call of `fs/copy-symlink`. This is less than **ms** but it's free **ms**.